dvc/command/data_sync.py

```diff
@@ -162,6 +164,12 @@ def add_parser(subparsers, _parent_parser):
         default=False,
         help="Pull cache for subdirectories of the specified directory.",
     )
+    pull_parser.add_argument(
+        "--clear-index",
+        action="store_true",
+        default=False,
+        help="Clear local index for the specified remote.",
```

Comment:
Desc doesn't match the other ones. Doesn't look like it was intended.

Reply:
Also, if we add it for `pull`, i guess we also have to add it for `fetch`. Hm...